client/scripts/controllers/chain/edgeware/signaling.ts

```diff
@@ -7,7 +7,7 @@ import { ProposalModule } from 'models';
 import SubstrateChain from 'controllers/chain/substrate/shared';
 import SubstrateAccounts, { SubstrateAccount } from 'controllers/chain/substrate/account';
 import { SubstrateCoin } from 'adapters/chain/substrate/types';
-import { SubstrateEntityKind } from 'events/substrate/types';
+import { SubstrateEntityKind } from '@commonwealth/chain-events/dist/src/substrate/types';
```

Comment:
why do we have `dist` here? Can it not import TS directly like we are doing with the API types?

Reply:
I can try messing with the exports / package.json config to fix that, we may need a top level index.ts file with all the imports together

Reply:
but yeah -- there is "no such thing" as a fully TS npm module. you have to compile it to .d.ts/.js pairs of types/implementation. this is just how the system works :\